PolyEngine/Editor/Src/Configs/ProjectConfig.cpp

```diff
@@ -40,19 +42,42 @@ String ProjectConfig::GetGameDllPath() const
 	return builder.StealString();
 }
 
+//------------------------------------------------------------------------------
 String ProjectConfig::GetRenderingDeviceDllPath() const
 {
 	StringBuilder builder;
 
 #if defined(_WIN32)
-	builder.Append(ProjectPath);
-	builder.Append("/Dist/Debug/PolyRenderingDeviceGL.dll");
+	builder.Append(GetDistDir());
+	builder.Append("PolyRenderingDeviceGL.dll");
 #elif defined(__linux__) || defined(__APPLE__)
-	builder.Append(ProjectPath)
-		.Append("/Dist/Debug/libPolyRenderingDeviceGL.so");
+	builder.Append(GetDistDir())
+		.Append("libPolyRenderingDeviceGL.so");
```

Comment:
Why? This is all in `AssetsPathConfig.json`. Extract it from there.

Reply:
Done